osu.Game/Screens/Edit/Editor.cs

```diff
@@ -538,12 +537,10 @@ protected override bool OnKeyDown(KeyDownEvent e)
                     // Seek to last object time, or track end if already there.
                     // Note that in osu-stable subsequent presses when at track end won't return to last object.
                     // This has intentionally been changed to make it more useful.
-                    double? lastObjectTime = editorBeatmap.HitObjects.LastOrDefault()?.GetEndTime();
-
-                    if (lastObjectTime == null || clock.CurrentTime == lastObjectTime)
+                    if (!editorBeatmap.HitObjects.Any() || clock.CurrentTime == editorBeatmap.GetLastObjectTime())
                         clock.Seek(clock.TrackLength);
                     else
-                        clock.Seek(lastObjectTime.Value);
+                        clock.Seek(editorBeatmap.GetLastObjectTime());
```

Comment:
If the condition in the `if` evaluates to `false`, then `GetLastObjectTime()` will be computed twice. Which seems wasteful given that the method is annotated as expensive.

Reply:
I did notice this, but i didn't see it as a huge concern. Will move out to a local.